fms2_io/netcdf_io.F90

```diff
@@ -404,6 +405,7 @@ function netcdf_file_open(fileobj, path, mode, nc_format, pelist, is_restart) &
   character(len=256) :: buf
   logical :: is_res
 
+  fileobj%is_root = .false.
```

Comment:
Won't this set the root to `.false.`?  This seems out of place for checking if the file is in append mode.

Reply:
No, all this does is initialize it to .false. 
fileobj%is_root is set on line 445 after getting the pelist and querying whether mpp_pe() matches the io_root.